sys-fabric/libibumad/libibumad-1.3.10.2.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="This is a sample skeleton ebuild file"
```

Comment:
Please fix the `DESCRIPTION=`. The original was good:
`DESCRIPTION="OpenIB User MAD lib functions which sit on top of the user MAD kernel modules"`